java/impl/aws/src/main/java/co/worklytics/psoxy/SQSBatchMergeHandler.java

```diff
@@ -0,0 +1,52 @@
+package co.worklytics.psoxy;
+
+import co.worklytics.psoxy.aws.AwsContainer;
+import co.worklytics.psoxy.aws.SQSOutput;
+import co.worklytics.psoxy.gateway.ProcessedContent;
+import co.worklytics.psoxy.gateway.impl.BatchMergeHandler;
+import co.worklytics.psoxy.aws.DaggerAwsContainer;
+
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Stream;
+
+/**
+ * @deprecated  - not sure we ultimately want/need this case, so marking as deprecated for now
```

Comment:
I think we can drop it, as the `WebhookCollectionModeHandler` will fit better. Other thing is if we'll need this in case of api v1/vpc stuff that is not *standard* in the configuration